packages/api-v2/src/hooks/useSortedMT5Accounts.ts

```diff
@@ -71,12 +82,17 @@ const useSortedMT5Accounts = (regulation?: string) => {
 
     // Sort the data by market_type to make sure the order is 'synthetic', 'financial', 'all'
     const sorted_data = useMemo(() => {
-        const market_type_order = ['synthetic', 'financial', 'all'];
+        const market_type_order = ['synthetic', 'financial', 'swap_free', 'zero_spread'];
 
         if (!filtered_data) return;
 
         const sorted_data = market_type_order.reduce((acc, market_type) => {
-            const accounts = filtered_data.filter(account => account.market_type === market_type);
+            const accounts = filtered_data.filter(account => {
+                if (account.market_type === 'all') {
+                    return account.product === market_type;
```

Comment:
product is the name of the product that we have and our main focus from now on would be on this, we won't be focusing on `market_type` anymore since removal of Business logic from FE. Previously we were checking market_type and sub_account_type to differentiate between accounts with `market_type = all`. With addition of `product` these kind of business logics can be removed from FE.